t/issue/1401.t

```diff
@@ -0,0 +1,86 @@
+
+use Test::More tests => 6;
+
+use strict;
+use warnings;
+use 5.010;
+use autodie qw(:all);
+use English qw($OSNAME -no_match_vars);
+use File::Temp;
+
+use Rex::Commands::Run;
+
+$::QUIET = 1;
+
+my $win = $OSNAME =~ m/^MSWin/mxsi;
+my $quote = q{"};
+my $space = q{ };
+my $empty = q{};
+
+sub command {
+  my ( $path, $exe, @parm ) = @_;
+  return qq{$path/$exe } . join $empty, @parm;
+}
+
+sub relative {
+  return qq{./$_[0]};
+}
+
+my $path    = File::Temp->newdir( 'path with spaces XXXX', DIR => q{./} );
+my $tail    = 'hello';
+my $parm    = 'Rex is wonderfull';
```

Comment:
I find `$parm` harder to read than it should be; I associate it with parmesan and parma ham, not with param(eter), or argument :D

Since this is used as an argument for our testing commands, perhaps `$argument` would be a better name for readability.